docs/whatsnew/1.1.0/1_1_0.rst

```diff
@@ -0,0 +1,55 @@
+Pylake 1.1.0
+============
+
+.. only:: html
+
+Pylake `v1.1.0` has been released. Once again, we'll use this section to highlight some of the new features and an important bugfix.
+
+Global analysis
+---------------
+
+Previously, downstream Kymograph analyses were limited to a single kymograph.
+With global analysis, one can combine tracking results from multiple kymographs to get better statistics when estimating dwell times or calculating diffusion constants.
+For more information on this feature, please refer to the relevant section of the :ref:`kymotracking tutorial<global_analysis>`.
+
+.. figure:: global_analysis.png
+
+    Global analysis allows you to analyze multiple kymographs in a single analysis.
+    This results in improved estimates of the biophysical parameters of your experiment.
+
+Bugfix sample_from_image
+------------------------
+
+Prior to version `1.1.0`, the method :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` had a bug that assumed the origin of a pixel to be at the edge rather than the center of the pixel.
+Consequently, the sampled window would be off by one pixel half the time.
+Note that the bug does not impact the position estimates.
+
+The old (incorrect) behavior is maintained until the next major release (2.0) to ensure backward compatibility and allow assessment of the impact of the bug.
+For reasonably sized sampling windows (more than 4 pixels wide), the effect of this bug should be small.
+When using :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` or :func:`~lumicks.pylake.KymoWidgetGreedy`, it is recommended to include the argument `correct_origin=True` which results in using the correct origin.
```

Comment:
I think we should put this in a warning box or something to really highlight it. Or we could try out one of the other boxes. 

I would move the sentence beginning with "For reasonably sized..." up to the previous paragraph before "Note that the...."
```suggestion
.. important:
    The old (incorrect) behavior is maintained until the next major release (2.0) to ensure backward compatibility and allow assessment of the impact of the bug.
    In order to apply this bugfix, include the argument `correct_origin=True` when using :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` or 
    :func:`~lumicks.pylake.KymoWidgetGreedy`
```

Reply:
Good call! I've put it in an important box now.